tasks/gulp/webjar-helpers.js

```diff
@@ -120,7 +125,7 @@ const webJarHelpers = ({
     </scm>
 
     <dependencies>
```

Comment:
I'm curious to understand if this is going to cause projects to pull in the webjar for govuk-frontend and accessible-autocomplete that duplicates what we're packaging into the hmrc-frontend webjar, or if this is just descriptive. It seems dangerous if it pulls them in as dependencies because webjars put stuff onto the classpath so you could theoretically get some unexpected runtime stuff happening if there was overlap in paths between different webjars. I guess it won't cause a problem if there isn't a webjar version built - since we stopped publishing govuk-frontend webjar versions but the new versions since then don't seem to have caused a problem

Reply:
@oscarduignan Agree, we should revisit this and see if the webjars are needed. Note we do publish the `govuk-frontend` webjar manually at the moment using npm webjars site. I propose creating a separate ticket to refactor this.

Reply:
yup separate ticket sounds right to me